### PR TITLE
Create LICENCE

### PR DESCRIPTION
**Pull request by Primeeight**

### CODE REVIEW

File: LICENCE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) [2023] [Stratego]
```
Comment (TylerPopson) [inline LICENCE:3]: The name is Statego, unfortunately we do not have the copyright to the game Stratego

**Review by TylerPopson — CHANGES_REQUESTED** [review]
The name of the app is wrong and should be Statego, not Stratego

**Comment by Primeeight** [comment]
Fixed App Name

**Review by TylerPopson — APPROVED** [review]
Looks good

**Comment by snappernapper07** [comment]
Hey I noticed that LICENSE is misspelled, and I know Sexton said that the file names really mattered. So if you could just fix this real quick, I think it would be for the best. Thanks!

**Pull request merged**